### DISCUSSION TITLE
Add more permissive experimental branches

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4695] [General Discussions]
Now adding a new feature is a big responsibility.
But it also looks like it's starting to become a bottleneck to a certain extent. 

I suggest creating some kind of experimental branches, where considering, reviewing and adding new features would not be so demanding and users  could download builds and test them (of course at their own peril and risk). 

I think with this approach we would get more feedback, we would have to guess less about whether a feature is needed or not. I think it would be worth it to have such a branch, although I understand that keeping this branch also requires some effort.

**Comment by Calinou** [comment]
I don't see how adding more branches to the godotengine/godot repository would help speed up development. You can already [download builds for a given pull request](https://docs.godotengine.org/en/latest/community/contributing/testing_pull_requests.html), or check out a pull request's branch and build it locally.

**Comment by ElliotBak3r** [comment]
There is the Goost project that sorts of fit this role if you would like to check it out
